docs/rest-api/source/includes/wp-api-v3/introduction.md

```diff
@@ -0,0 +1,82 @@
+# Introduction
+
+WooCommerce (WC) 2.6+ is fully integrated with the WordPress [REST](http://en.wikipedia.org/wiki/Representational_State_Transfer) API. This allows WC data to be created, read, updated, and deleted using requests in JSON format and using WordPress REST API Authentication methods and standard HTTP verbs which are understood by most HTTP clients.
+
+The current WP REST API integration version is `v3` which takes a first-order position in endpoints.
+
+## Requirements
+
+To use the latest version of the REST API you must be using:
+
+-   WooCommerce 3.5+.
+-   WordPress 4.4+.
```

Comment:
I know we have a compatibility file somewhere. Could it be possible to somehow automate that these versions are pulled from there or updated? 

My general worry with the approach is that documentation will go out of date unless some strong practises are put in place.

Reply:
You're right. At the moment, I'm not sure of a foolproof solution. Perhaps we can revisit this later?